timeserieskmeans_final.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+"""TimeSeriesKMeans
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1gMoRoACt_ai41MktAwIxyq38Q0ic4bv-
+"""
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+from tslearn.metrics import dtw
+from tslearn.metrics import soft_dtw
+from tslearn.clustering import TimeSeriesKMeans
+
+def kmeans(csv,k,k_seeds,xaxis, yaxis, x_lower=None,x_upper=None,y_lower=None,y_upper=None):
+  df = pd.read_csv(f'{csv}')
+  df = df.dropna()
+
+  df['Day'] = df.index +1
+
+
+  #assuming first column is the datetime column
```

Comment:
Jae has an example CSV on google drive but regardless, all we are doing is importing the CSV directly into a dataframe with read_csv(). Use the timeseries column from that